crates/viewer/re_ui/src/ui_layout.rs

```diff
@@ -0,0 +1,136 @@
+/// Specifies the context in which the UI is used and the constraints it should follow.
+#[derive(Clone, Copy, Debug, PartialEq, Eq)]
+pub enum UiLayout {
```

Comment:
This has just moved from `crates/viewer/re_viewer_context/src/component_ui_registry.rs`